src/redux/reducer.test.ts

```diff
@@ -7,22 +7,28 @@
 
 // Because of a circular import issue, we have to import the store to run the
 // unit tests, even if your IDE is showing that the import is unused.
-// eslint-disable-next-line
-import { store } from './store';
-import { reducer } from './reducer';
 import {
-    resetNetworkAreaDiagramDepth,
-    incrementNetworkAreaDiagramDepth,
-    decrementNetworkAreaDiagramDepth,
-    setFullScreenDiagram,
-    openDiagram,
-    minimizeDiagram,
-    togglePinDiagram,
+    Actions,
+    AppState,
+    initialState,
```

Comment:
Not needed

Reply:
Yes, I've see it when I updated the branch with main, it broke tests.